lib/exgboost.ex

```diff
@@ -163,38 +130,38 @@ defmodule EXGBoost do
   end
 
   @doc """
-  Train a new booster model given a data tensor and a label tensor
+  Train a new booster model given a data tensor and a label tensor.
 
   ## Options
 
-  * `:obj` - Specify the learning task and the corresponding learning objective. This function must accept two arguments: preds, dtrain. preds is an array of predicted real valued scores. dtrain is the training data set. This function returns gradient and second order gradient.
+    * `:obj` - Specify the learning task and the corresponding learning objective. This function must accept two arguments: preds, dtrain. preds is an array of predicted real valued scores. dtrain is the training data set. This function returns gradient and second order gradient.
```

Comment:
I can reword the parameters to combine :obj and :objective into the same param for the next release